File: Nested for Pattern 4/code4.c
#include <stdio.h>

void main (){

	int row;

	printf("Enter no. of rows : " );
	scanf("%d",&row);

	for(int i = 1; i <= row ; i++ ){

		int count = i;
		for(int j = 0 ; j < row ; j++){

			printf("%d	",count);
			if(count == row){

				count = 0;
			}
			count ++;


		}

		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/11/code4.java

import java.util.Scanner;

class SpaceDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.print("Enter rows : ");
		int row = s.nextInt();

		for(int i = 0 ; i < row ; i++){

			for(int k = 0 ; k < row - i - 1; k++){

				System.out.print("		");
			}

			for(int j = 0; j <= i ; j++){

				System.out.print(row*(j+1)+"		");


			}

				System.out.print("\n\n");

		}

	}

}
